pkgs/development/tools/kustomize/default.nix

```
@@ -1,29 +1,29 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub, tree }:
 
 buildGoModule rec {
   pname = "kustomize";
-  version = "3.1.0";
-  # rev is the 3.1.0 commit, mainly for kustomize version command output
-  rev = "95f3303493fdea243ae83b767978092396169baf";
+  version = "3.3.1";
+  # rev is the 3.3.1 commit, mainly for kustomize version command output
+  # rev = "f2ac5a2d0df13c047fb20cbc12ef1a3b41ce2dad";
```

in otder to make suggestion below work you need to uncomment this.

```suggestion
  rev = "f2ac5a2d0df13c047fb20cbc12ef1a3b41ce2dad";
```